Mathlib/Topology/UniformSpace/Basic.lean

```diff
@@ -1259,6 +1259,7 @@ instance inhabitedUniformSpaceCore : Inhabited (UniformSpace.Core α) :=
 
 /-- Given `f : α → β` and a uniformity `u` on `β`, the inverse image of `u` under `f`
   is the inverse image in the filter sense of the induced function `α × α → β × β`. -/
+@[reducible]
```

Comment:
Can you add a reference to "reducible non-instances" here?

Reply:
Done